app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
I would remove registerable so that access is only permitted by invite... In the short term we have no plans of actually maintaining devise users since this will probably only be an read only API endpoint.